Lib/test/test_minidom.py

```diff
@@ -77,6 +77,51 @@ def testParseFromTextFile(self):
             dom.unlink()
             self.confirm(isinstance(dom, Document))
 
+    def testAttrModeSetsParamsAsAttrs(self):
+        attr = Attr("qName", "namespaceURI", "localName", "prefix")
+        self.assertEqual(
+            attr._name,
```

Comment:
I don't think we need to test the private attributes, just test `attr.name`. This also goes for a few other tests.

Reply:
Since we are explicitly adding tests related to the failure to test private attributes, isn't that exactly what we don't want to do?

Reply:
But we care about the effect of the private attributes on public APIs, not about the private attributes themselves. In this case, we can simply test `attr.localName` to make sure the `localName` argument is respected.

Reply:
I'm a bit torn by that argument, but not worth the hassle of discussing.